lnrpc/lightning.proto

```diff
@@ -3201,6 +3202,7 @@ message Invoice {
     A bare-bones invoice for a payment within the Lightning Network. With the
     details of the invoice, the sender has all the data necessary to send a
     payment to the recipient.
+    Note: Output only, don't specify for creating an invoice.
```

Comment:
I think we should also add the comment for the `creation_date`, `settle_date`, 'settle_index', 'amt_paid_sat', 'amt_paid_msat', 'state', 'htlcs', 'features', 'is_keysend' and 'amp_invoice_state'.